File: Salonify.Api/models/SalonService.cs
using MongoDB.Bson.Serialization.Attributes;

public class SalonService
{
    [BsonRepresentation(MongoDB.Bson.BsonType.String)]
    public ServiceType ServiceType { get; set; }
    public string Name{ get; set; }
    public string Description { get; set; }

    public decimal Price { get; set; }

    public int DurationMinutes { get; set; }

    public string ImageUrl { get; set; }
}

File: SalonifyFrontend/salonifyfrontend/types/SalonService.ts

export type SalonService = {
  serviceType: number;
  name: string;
  description?: string;
  price: number;
  durationMinutes: number;
  imageUrl?: string | null;
  serviceName?: string;
};

export type AvailableService = {
  serviceType: number | string;
  name?: string;
  serviceName?: string;
};